tests/test_utils_pytorch_numpy_unification.py

```diff
@@ -27,6 +27,13 @@
     TEST_MODE.append([p(np.array([3.1, 4.1, 4.1, 5.1])), p(4.1), False])
     TEST_MODE.append([p(np.array([3.1, 4.1, 4.1, 5.1])), p(4), True])
 
+TEST_MIN_MAX = []
+for p in TEST_NDARRAYS:
+    TEST_MIN_MAX.append([p(np.array([1, 2, 3, 4, 4, 5])), {}, min, p(1)])
```

Comment:
Hi @KumoLiu , thanks for the fix and it looks good me. My only question is if we should cover torch array in the test cases?

Reply:
all test cases are covered by using `TEST_NDARRAYS`.
https://github.com/Project-MONAI/MONAI/blob/cea80a686b907b22d90c8024cea5a17ef9d9f58a/tests/utils.py#L839

Reply:
I see. Thanks!